src/components/vm/nics/nicAdd.tsx

```diff
@@ -151,24 +155,32 @@ export class AddNIC extends React.Component<AddNICProps, AddNICState> {
 
         this.state = {
             dialogError: undefined,
-            networkType: "network",
+            networkType: this.props.vm.connectionName == "session" ? "user" : "network",
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-66f0c2f5-20251104-070631-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)